openassessment/xblock/static/js/src/lms/oa_staff.js

```diff
@@ -0,0 +1,34 @@
+/**
+Interface for staff assessment view.
```

Comment:
Nit: can you give these well-formed JSDoc headers, rather than the strange hybrids that the ORA code uses.

http://usejsdoc.org/about-getting-started.html